src/python_testing/TC_RVCRUNM_2_2.py

```diff
@@ -38,17 +39,25 @@
 # --int-arg PIXIT.RVCRUNM.MODE_A:<mode id> PIXIT.RVCRUNM.MODE_B:<mode id>
 
 
+class RvcStatusEnum(enum.IntEnum):
+    # TODO remove this class once InvalidInMode response code is implemented in python SDK
```

Comment:
Please create an issue for this.

Reply:
I'm surprised this isn't already there.

Reply:
https://github.com/project-chip/connectedhomeip/issues/35031